src/util/mockDataUtils.ts

```diff
@@ -32,58 +33,66 @@ export const fakeEvents = (
 });
 
 export const fakeEvent = (overrides?: Partial<EventDetails>): EventDetails => {
-  return {
-    id: `hel:${faker.random.uuid()}`,
-    internalId: faker.random.uuid(),
-    name: fakeLocalizedObject(faker.name.title()),
-    publisher: 'provider:123',
-    provider: fakeLocalizedObject(),
-    shortDescription: fakeLocalizedObject(),
-    description: fakeLocalizedObject(),
-    images: [fakeImage()],
-    infoUrl: fakeLocalizedObject(),
-    inLanguage: [fakeInLanguage()],
-    audience: [],
-    keywords: [fakeKeyword()],
-    location: fakePlace(),
-    startTime: '2020-07-13T05:51:05.761000Z',
-    endTime: null,
-    datePublished: null,
-    externalLinks: [] as any,
-    offers: [] as any,
-    subEvents: [] as any,
-    eventStatus: 'EventScheduled',
-    superEvent: null,
-    dataSource: 'hel',
-    __typename: 'EventDetails',
-    ...overrides,
-  };
+  return merge(
```

Comment:
Why merge is needed?

Reply:
Using spread syntax for overrides caused me some extra work in cases when I wanted to replace only one field of nested object. 
For example if I want to override only name.fi not the complete name object.
```
const event = fakeEvent({
  name: { fi: eventName },
}) as EventFieldsFragment;
```

Reply:
OK, nice addition 👍